keps/sig-network/1672-tracking-terminating-endpoints/kep.yaml

```diff
@@ -0,0 +1,20 @@
+title: Tracking Terminating Endpoints in EndpointSlice
```

Comment:
@robscott Are you good with this?

Reply:
Added a couple small nits, but LGTM overall. This specific KEP feels like it might make sense as part of the existing [EndpointSlice KEP](https://github.com/kubernetes/enhancements/blob/master/keps/sig-network/20190603-endpointslices/README.md). That has traditionally been where the EndpointSlice API and any changes to it have been defined. With that said, I know that could result in it becoming a massive KEP. Not sure what the best practice is here.

Reply:
Tim had previously suggested that this should be a new KEP (see [here](https://github.com/kubernetes/enhancements/pull/1607#pullrequestreview-372462904)) and I agree. 

I think it would make sense for this KEP to summarize the motivation for the change but we should still backport the API changes into the main EPSlice KEP with maybe a reference to this one?

Reply:
(I'll open a follow-up PR for this)